ibis/backends/datafusion/compiler.py

```diff
@@ -102,6 +103,8 @@ def visit_NonNullLiteral(self, op, *, value, dtype):
             return self.f.date_trunc("day", value.isoformat())
         elif dtype.is_binary():
             return sg.exp.HexString(this=value.hex())
+        elif dtype.is_uuid():
+            return sge.convert(str(value))
```

Comment:
This is the same in the pyspark backend. Could this be moved to the base compiler perhaps?

Reply:
A bunch of the backends have this implementation, but I'd rather not do this refactor now (or maybe ever). I don't think we lose much by repeating this particular 2 lines.